prestart.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+sleep 10;
+createsuperuser --email=$EMAIL --password=$PASSWORD --full-name=admin
```

Comment:
```suggestion
sleep 10
createsuperuser --email="$EMAIL" --password="$PASSWORD" --full-name=admin
```

Using `;` is not necessary at end of lines in bash

Reply:
We should also use `${foo}` for good practice as noted here https://stackoverflow.com/a/8748880